README.md

```diff
@@ -1580,7 +1579,7 @@ imports:
 zenstruck_foundry:
 
     # Whether to auto-refresh proxies by default (https://github.com/zenstruck/foundry#auto-refresh)
-    auto_refresh_proxies: false
+    auto_refresh_proxies: null # (defaults to false but will default to true in 2.0)
```

Comment:
How else should we document that this "needs" to be configured?

Reply:
I would just say `auto_refresh_proxies: true`. The rest is implementation detail to me, and the code/deprecation message do a good enough job to help the user